Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.mm

```diff
@@ -435,7 +436,7 @@
         triggerRenderingUpdate();
         m_deferredRenderingUpdateWhileWaitingForBackingStoreSwap = false;
         m_isScheduled = false;
-    } else
+    } else if (!wasWaitingForBackingStoreSwap || !m_updateRenderingTimer.isActive())
```

Comment:
Hard to know if this is correct

Reply:
I've reversed the test (much easier to read), and I've added an action to set `m_deferredRenderingUpdateWhileWaitingForBackingStoreSwap`, see commit description for rationale.